examples/tracing_loops.py

```diff
@@ -0,0 +1,63 @@
+"""
+=====================
+Coronal Loops Tracing
+=====================
+
+This example applies traces out the coronal loops in a FITS image file
+using `sunkit_image.trace.occult2`. In this example we will use the settings
+and the data from Markus J. Aschwanden tutorial on his IDL implemetation of
+the ``OCCULT2`` algorithm, which can be found `here <http://www.lmsal.com/~aschwand/software/tracing/tracing_tutorial1.html>__.
+The aim of this example is to demonstrate that `sunkit_image.trace.occult2` provides similar
+results as the IDL version.
+"""
+# sphinx_gallery_thumbnail_number = 2
+
+import matplotlib.pyplot as plt
+import astropy
+import numpy as np
+
+import sunkit_image.trace as trace
+
+###########################################################################
+# We will be using `astropy.io.fits.getdata` to read a FITS file from the tutorial website
+image = astropy.io.fits.getdata("http://www.lmsal.com/~aschwand/software/tracing/TRACE_19980519.fits", ignore_missing_end=True)
+
+# The original image is plotted to showcase the difference. Matplotlib plots are inverted by default so
+# to fix it we will be using ``origin`` as ``lower``.
+plt.imshow(image, cmap="Spectral", origin="lower")
```

Comment:
```suggestion
plt.imshow(image, cmap="Spectral", origin="lower")
```

Is that the correct colour map? @wafels, do we have the colormap in SunPy?

Reply:
I am not sure this was the best one which I found, if there is something better we can change it

Reply:
http://docs.sunpy.org/en/stable/code_ref/cm.html

I think its the top colour map here.